### ISSUE TITLE
fluent breaks doc build

### ISSUE THREAD

**Issue by asmagill** [Hammerspoon/hammerspoon#351]
With latest pull (which includes fluent), the following error occurs during doc building process:

~~~bash
scripts/docs/bin/gensql < build/docs.json | sqlite3 build/docs.sqlite
Error: near line 537: UNIQUE constraint failed: searchIndex.name, searchIndex.type, searchIndex.path
make: *** [build/docs.sqlite] Error 1
~~~

Tracked this down to what appears to be a duplicate definition for:

~~~sql
INSERT INTO searchIndex VALUES (NULL, 'hs.window.CoordTrans', 'Method', 'hs.window.html#CoordTrans');
INSERT INTO searchIndex VALUES (NULL, 'hs.window.CoordTrans', 'Method', 'hs.window.html#CoordTrans');
~~~

And further searching puts that doc label in fluent:

~~~bash
$ ack CoordTrans extensions/
extensions/window/fluent.lua:51:-- grids, by passing a right CoordTrans object.
extensions/window/fluent.lua:52:local CoordTrans = {}
extensions/window/fluent.lua:54:--- hs.window.CoordTrans:get(win[, scr]) -> table
extensions/window/fluent.lua:68:function CoordTrans:get(win, scr)
extensions/window/fluent.lua:83:--- hs.window.CoordTrans:set(win, screen, f) -> nil
extensions/window/fluent.lua:94:function CoordTrans:set(win, screen, f)
extensions/window/fluent.lua:106:-- instantiates a CoordTrans object
extensions/window/fluent.lua:109:  setmetatable(self, { __index = CoordTrans })
extensions/window/fluent.lua:118:---  * ct - CoordTrans object
extensions/window/fluent.lua:124:--- * CoordTrans is an object though which one gets coordinates for a window on the
extensions/window/fluent.lua:126:---   on a screen. See CoordTrans description.
extensions/window/fluent.lua:177:---  * ct - CoordTrans object
extensions/window/fluent.lua:183:--- * CoordTrans is an object though which one gets coordinates for a window on the
extensions/window/fluent.lua:185:---   on a screen. See CoordTrans description.
extensions/window/fluent.lua:198:---  * ct - CoordTrans object
extensions/window/fluent.lua:204:--- * CoordTrans is an object though which one gets coordinates for a window on the
extensions/window/fluent.lua:206:---   on a screen. See CoordTrans description.
~~~

I can try and look into this deeper later, but wanted to get it out there in case anyone has time to fix it sooner.

**Comment by cmsj** [comment]
one of these days I will remember to test "make docs" before I properly merge something.

**Comment by cmsj** [comment]
This should be fixed by 3db85ef
